linked-wiki-pages/vss-extension.json

```diff
@@ -28,12 +29,12 @@
       "properties": {
         "name": "Linked wiki pages",
         "uri": "linked-wiki-pages.html",
-        "height": 200
+        "height": 80
       }
     }
   ],
   "categories": [
-    "Plan and track"
```

Comment:
Plan and track is now deprecated, might be another more fitting category-name: https://docs.microsoft.com/en-us/azure/devops/extend/develop/manifest?view=azure-devops